packages/roosterjs-editor-api/lib/format/createLink.ts

```diff
@@ -54,7 +55,7 @@ export default function createLink(
     displayText?: string
 ) {
     editor.focus();
-    let url = link ? link.trim() : '';
+    let url = (checkXss(link) || '').trim();

```

Comment:
Add back the check to make sure link is defined? `checkXss `should work anyways, but would prevent a bit of unnecessary work

Reply:
I think they both work, no difference.